apps/heft/.npmignore

```diff
@@ -33,4 +33,4 @@
 
 !/includes/**
 !UPGRADING.md
-
+lib-*/root-lib/
```

Comment:
This file accepts comments, so put a file about what the heck this pattern is matching